user/decibels/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "decibels"
+pkgver = "46.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "blueprint-compiler",
+    "desktop-file-utils",
+    "gettext",
+    "libxml2-progs",
+    "meson",
+    "pkgconf",
+    "typescript",
+]
+makedepends = ["gjs-devel"]
+depends = ["gjs", "libadwaita"]
```

Comment:
also gst-plugins-good i think (not sure which but something gst)

Reply:
from the GI definitions it pulls, apparently it only depends on gst-plugins-base